template/template.go

```
@@ -33,6 +35,64 @@ type TemplateConfiguration struct {
 	configuration.Installation
 }
 
+// TemplateHelmChart takea a filesystem, a path to a directory containing a
+// helm chart, and a BuildInfo struct.
+// It templates the chart's Chart.yaml and templates/deployment.yaml
+// with this information.
+// It is an error if there are multiple charts in the helm directory.
+func TemplateHelmChart(fs afero.Fs, helmPath string, buildInfo BuildInfo) error {
+	fileInfos, err := afero.ReadDir(fs, helmPath)
+	if err != nil {
+		return err
```

Please mask the errors.